README.md

```diff
@@ -161,10 +161,12 @@ A curated list of awesome Java frameworks, libraries and software.
 * [H2](http://h2database.com/) - Small SQL Database notable for its in-memory functionality.
 * [HikariCP](https://github.com/brettwooldridge/HikariCP) - High performance JDBC connection pool.
 * [JDBI](http://jdbi.org/) - Convenient abstraction of JDBC.
+* [Jedis](https://github.com/xetorthio/jedis) - A small and sane client for redis
```

Comment:
Could you please add the punctuation and rewrite the description? Something like "Client for Redis." is the minimum, although it would be better if you could add some strong points.